pkg/crc/init.go

```diff
@@ -0,0 +1,15 @@
+package crc
+
+import (
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/state"
+)
+
+func init() {
```

Comment:
When is this `init()` function called?

Reply:
on application start... when the library is loaded the first time (so before any function is called that is part of this library)